src/common/Exporter.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define([], factory);
+    } else {
+        root.common_Exporter = factory();
+    }
+}(this, function () {
+    function Exporter(widget) {
+        this._widget = widget;
+    }
+
+    Exporter.prototype.formatData = function (_) {
```

Comment:
@GordonSmith Do you mean get rid of Exporter.js and just put the formatting logic in Widget.js as Widget.prototype.export() and the download function in Utility.js?

If that is not what you mean, please elaborate.